plugins/catalog-react/src/components/EntityOwnerPicker/useFacetsEntities.ts

```diff
@@ -43,6 +52,8 @@ type FacetsInitialRequest = {
  */
 export function useFacetsEntities({ enabled }: { enabled: boolean }) {
   const catalogApi = useApi(catalogApiRef);
+  const apis = useApiHolder();
+  const entityPresentationApi = apis.get(entityPresentationApiRef);
```

Comment:
What's the motivation for making `entityPresentationApi` optional here?

Can we just do `useApi(entityPresentationApiRef)` instead and make it required?

It should cleanup some other code further down too